packages/core/admin/server/bootstrap.js

```diff
@@ -94,4 +95,6 @@ module.exports = async () => {
 
   apiTokenService.checkSaltIsDefined();
   tokenService.checkSecretIsDefined();
+
+  await bootstrapDataTransfer(strapi);
```

Comment:
@Convly Opinions on this versus setting it up to be able to do `getService('data-transfer').bootstrap()`?